lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -391,7 +512,8 @@ void SessionStateEnforcer::m5g_update_session_state_to_amf(
  */
 void SessionStateEnforcer::prepare_response_to_access(
     const std::string& imsi, SessionState& session_state,

```

Comment:
Could we just pass SessionContext here instead of the full SessionState?

Reply:
adopted